modules/veteran/spec/requests/v0/accredited_representatives_spec.rb

```diff
@@ -126,112 +126,6 @@
       end
     end
 
-    context 'when there are no results for the search criteria' do
-      it 'returns an empty list' do
-        get '/services/veteran/v0/accredited_representatives',
-            params: { type: 'organization', lat: 40.7128, long: -74.0060 }
-
-        parsed_response = JSON.parse(response.body)
-
-        expect(parsed_response['data']).to eq([])
-        expect(parsed_response['meta']['pagination']['total_entries']).to eq(0)
-      end
-    end
-
-    it 'returns ok for a successful request' do
-      get '/services/veteran/v0/accredited_representatives',
-          params: { type: 'organization', lat: 40.7128, long: -74.0060 }
-
-      expect(response).to have_http_status(:ok)
-    end
```

Comment:
These two specs are still valuable as they describe behavior that is still in the controller. Just need to change `organization` to `attorney`.

Reply:
Okay, thanks.

Reply:
Thanks, Jon. I added them back.